src/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { LinkButton } from '../../components';
+import { Menu, Dropdown, Typography, Row, Col, Button } from 'antd';
+import { DownOutlined, UserOutlined } from '@ant-design/icons';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+const { Text } = Typography;
+
+const NavRow = styled(Row)`
+  background-color: white;
+  padding: 1em 0;
+`;
+const AntButtonLink = styled(LinkButton)`
+  color: black;
+  margin-left: 7em;
+`;
+const Logo = styled.img`
+  float: right;
+`;
+const LogoText = styled(Text)`
```

Comment:
Probably isn't necessary for the scaffold, but I would move all typography to that styles folder and always use your typography styled components for any text. Keeps the site more consistent.

Reply:
I might just remove most of the styling @anirudh723 did for the navbar for the frontend scaffold then